Source/Modules/ModuleScene.cpp

```diff
@@ -365,3 +356,51 @@ std::vector<GameObject*> ModuleScene::CreateHierarchyFromJson(Json& jsonGameObje
 	}
 	return loadedObjects;
 }
+
+
+void ModuleScene::AddGameObjectAndChildren(GameObject* object)
+{
+	AddGameObject(object);
+
+	for (GameObject* children : object->GetChildren())
+	{
+		AddGameObjectAndChildren(children);
+	}
+
+}
+
+void ModuleScene::RemoveGameObjectAndChildren(GameObject* object)
+{
+	RemoveGameObject(object);
+
+	for (GameObject* children : object->GetChildren())
```

Comment:
```suggestion
	for (GameObject* child: object->GetChildren())
```
This is more readable